samples/tab-deeplink-all-features/csharp/DeeplinkAllFeatures/wwwroot/js/DeeplinkAllFeatures.js

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// App Initilization
+microsoftTeams.app.initialize();
+
+// Select People for audio and video call.
+function selectPeople() {
+    microsoftTeams.app.initialize().then(() => {
+        microsoftTeams.people.selectPeople({ setSelected: [], openOrgWideSearchInChatOrChannel: true, singleSelect: false }).then((people) => {
+            if (people) {
+                document.getElementById("Attendee").value = (people.map((p) => p.email)).join(",");
+            }
+        }).catch((error) => {
+            console.log(error);
+        });
+    });
+}
+
+// function to make audio and video calls 
+function onCallDeepLinkButtonClick(callModalities) {
+    microsoftTeams.app.initialize().then(() => {
+        microsoftTeams.people.selectPeople({ setSelected: [], openOrgWideSearchInChatOrChannel: true, singleSelect: false }).then((people) => {
+            if (people) {
+                if (microsoftTeams.call.isSupported()) {
+                    microsoftTeams.call.startCall({
+                        targets: people.map((p) => p.email),
+                        requestedModalities: [callModalities]
+                    })
+                        .then((result) => {
+                            console.log(result);
+                        }).catch((error) => {
+                            console.log(error);
+                        });
+                }
+                else {
+                    alert("Call isn't supported");
+                }
+            }
+        }).catch((error) => {
+            console.log(error);
+        });
+    });
+}
+
+// Opens aplication profile dialog. 
+function onApplicationProfileDialog() {
+    let appId = "<<Your_App_ID>>"; // you will get app-Id from teams admin portal, app profile page.
```

Comment:
Read it from .env file otherwise deployment will fail.

Reply:
ok